.luna-package/Dataframes.lunaproject

```diff
@@ -41,8 +41,8 @@ modulesSettings:
     breadcrumbsSettings:
     - - items: []
       - breadcrumbCameraSettings:
-          logicalToScreen: ! '[[1.0,0.0,0.0,0.0],[0.0,1.0,0.0,0.0],[0.0,0.0,1.0,0.0],[-344.0,83.0,0.0,1.0]]'
-          screenToLogical: ! '[[1.0,0.0,0.0,0.0],[0.0,1.0,0.0,0.0],[0.0,0.0,1.0,0.0],[344.0,-83.0,0.0,1.0]]'
+          logicalToScreen: ! '[[1.0,0.0,0.0,0.0],[0.0,1.0,0.0,0.0],[0.0,0.0,1.0,0.0],[-264.0,-24.0,0.0,1.0]]'
```

Comment:
I think this file should be restored to some "clean, default" state and added to .gitignore so further local changes are not versioned by default.